website/content/en/docs/installation/install-operator-sdk.md

```diff
@@ -24,87 +23,60 @@ description: Install the Operator SDK CLI
-$ RELEASE_VERSION=v1.1.0
 # Linux
-$ curl -LO https://github.com/operator-framework/operator-sdk/releases/download/${RELEASE_VERSION}/operator-sdk-${RELEASE_VERSION}-x86_64-linux-gnu
-$ curl -LO https://github.com/operator-framework/operator-sdk/releases/download/${RELEASE_VERSION}/ansible-operator-${RELEASE_VERSION}-x86_64-linux-gnu
-$ curl -LO https://github.com/operator-framework/operator-sdk/releases/download/${RELEASE_VERSION}/helm-operator-${RELEASE_VERSION}-x86_64-linux-gnu
+curl -Lo operator-sdk https://github.com/operator-framework/operator-sdk/releases/latest/download/operator-sdk_"$(uname -m)"_linux
 # macOS
-$ curl -LO https://github.com/operator-framework/operator-sdk/releases/download/${RELEASE_VERSION}/operator-sdk-${RELEASE_VERSION}-x86_64-apple-darwin
-$ curl -LO https://github.com/operator-framework/operator-sdk/releases/download/${RELEASE_VERSION}/ansible-operator-${RELEASE_VERSION}-x86_64-apple-darwin
-$ curl -LO https://github.com/operator-framework/operator-sdk/releases/download/${RELEASE_VERSION}/helm-operator-${RELEASE_VERSION}-x86_64-apple-darwin
+curl -Lo operator-sdk https://github.com/operator-framework/operator-sdk/releases/latest/download/operator-sdk_"$(uname -m)"_darwin
 ```
 
 #### Verify the downloaded release binaries
 
 ```sh
-# Linux
-$ curl -LO https://github.com/operator-framework/operator-sdk/releases/download/${RELEASE_VERSION}/operator-sdk-${RELEASE_VERSION}-x86_64-linux-gnu.asc
-$ curl -LO https://github.com/operator-framework/operator-sdk/releases/download/${RELEASE_VERSION}/ansible-operator-${RELEASE_VERSION}-x86_64-linux-gnu.asc
-$ curl -LO https://github.com/operator-framework/operator-sdk/releases/download/${RELEASE_VERSION}/helm-operator-${RELEASE_VERSION}-x86_64-linux-gnu.asc
-# macOS
-$ curl -LO https://github.com/operator-framework/operator-sdk/releases/download/${RELEASE_VERSION}/operator-sdk-${RELEASE_VERSION}-x86_64-apple-darwin.asc
-$ curl -LO https://github.com/operator-framework/operator-sdk/releases/download/${RELEASE_VERSION}/ansible-operator-${RELEASE_VERSION}-x86_64-apple-darwin.asc
-$ curl -LO https://github.com/operator-framework/operator-sdk/releases/download/${RELEASE_VERSION}/helm-operator-${RELEASE_VERSION}-x86_64-apple-darwin.asc
+curl -LO https://github.com/operator-framework/operator-sdk/releases/latest/download/checksums.txt
 ```
 
 To verify a release binary using the provided asc files, place the binary and corresponding asc file into the same directory and use the corresponding command:
 
 ```sh
-# Linux
-$ gpg --verify operator-sdk-${RELEASE_VERSION}-x86_64-linux-gnu.asc
-$ gpg --verify ansible-operator-${RELEASE_VERSION}-x86_64-linux-gnu.asc
-$ gpg --verify helm-operator-${RELEASE_VERSION}-x86_64-linux-gnu.asc
-# macOS
-$ gpg --verify operator-sdk-${RELEASE_VERSION}-x86_64-apple-darwin.asc
-$ gpg --verify ansible-operator-${RELEASE_VERSION}-x86_64-apple-darwin.asc
-$ gpg --verify helm-operator-${RELEASE_VERSION}-x86_64-apple-darwin.asc
+gpg --verify ./operator-sdk
```

Comment:
These changes aren't correct, still need to figure out a flow for verifying release binaries.